python-sdk/pachyderm_sdk/api/pfs/extension.py

```diff
@@ -231,7 +255,7 @@ def put_file_from_file(
         self._stub.put_file_from_file(commit=self, path=path, file=file, append=append)
         return File(commit=self._commit, path=path)
 
-    def copy_file(self, *, src: "File", dst: str, append: bool = True) -> "File":
+    def copy_file(self, *, src: "File", dst: str, append: bool = False) -> "File":
```

Comment:
Strange that the default for copy was append

Reply:
This was a mistake when moving this from python_pachyderm. It's False there, I'm not sure how these got flipped.